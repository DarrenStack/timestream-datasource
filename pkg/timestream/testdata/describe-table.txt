Frame[0] {
    "custom": {
        "queryId": "{CHANGES}"
    },
    "stats": []
}
Name: 
Dimensions: 3 Fields by 15 Rows
+-------------------+----------------+-----------------------+
| Name: Column      | Name: Type     | Name: Column Category |
| Labels:           | Labels:        | Labels:               |
| Type: []string    | Type: []string | Type: []string        |
+-------------------+----------------+-----------------------+
| availability_zone | varchar        | DIMENSION             |
| microservice_name | varchar        | DIMENSION             |
| instance_name     | varchar        | DIMENSION             |
| process_name      | varchar        | DIMENSION             |
| os_version        | varchar        | DIMENSION             |
| jdk_version       | varchar        | DIMENSION             |
| cell              | varchar        | DIMENSION             |
| region            | varchar        | DIMENSION             |
| silo              | varchar        | DIMENSION             |
| ...               | ...            | ...                   |
+-------------------+----------------+-----------------------+



