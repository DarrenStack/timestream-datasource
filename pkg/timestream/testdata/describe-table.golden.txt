ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ

Frame[0] {}
Name: 
Dimensions: 3 Fields by 15 Rows
+-------------------+-----------------+-----------------------+
| Name: Column      | Name: Type      | Name: Column Category |
| Labels:           | Labels:         | Labels:               |
| Type: []*string   | Type: []*string | Type: []*string       |
+-------------------+-----------------+-----------------------+
| availability_zone | varchar         | DIMENSION             |
| microservice_name | varchar         | DIMENSION             |
| instance_name     | varchar         | DIMENSION             |
| process_name      | varchar         | DIMENSION             |
| os_version        | varchar         | DIMENSION             |
| jdk_version       | varchar         | DIMENSION             |
| cell              | varchar         | DIMENSION             |
| region            | varchar         | DIMENSION             |
| silo              | varchar         | DIMENSION             |
| ...               | ...             | ...                   |
+-------------------+-----------------+-----------------------+


====== TEST DATA RESPONSE (arrow base64) ======
FRAME=QVJST1cxAAD/////6AEAABAAAAAAAAoADgAMAAsABAAKAAAAFAAAAAAAAAEDAAoADAAAAAgABAAKAAAACAAAAHQAAAADAAAATAAAACgAAAAEAAAAoP7//wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAADA/v//CAAAAAwAAAAAAAAAAAAAAAQAAABuYW1lAAAAAOD+//8IAAAADAAAAAIAAAB7fQAABAAAAG1ldGEAAAAAAwAAANwAAABwAAAABAAAAEL///8UAAAARAAAAEQAAAAAAAUBQAAAAAEAAAAEAAAAMP///wgAAAAYAAAADwAAAENvbHVtbiBDYXRlZ29yeQAEAAAAbmFtZQAAAAAAAAAAMP///w8AAABDb2x1bW4gQ2F0ZWdvcnkAqv///xQAAAA8AAAAPAAAAAAABQE4AAAAAQAAAAQAAACY////CAAAABAAAAAEAAAAVHlwZQAAAAAEAAAAbmFtZQAAAAAAAAAAkP///wQAAABUeXBlAAASABgAFAATABIADAAAAAgABAASAAAAFAAAAEQAAABIAAAAAAAFAUQAAAABAAAADAAAAAgADAAIAAQACAAAAAgAAAAQAAAABgAAAENvbHVtbgAABAAAAG5hbWUAAAAAAAAAAAQABAAEAAAABgAAAENvbHVtbgAA/////xgBAAAUAAAAAAAAAAwAFgAUABMADAAEAAwAAACIAgAAAAAAABQAAAAAAAADAwAKABgADAAIAAQACgAAABQAAACoAAAADwAAAAAAAAAAAAAACQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAQAAAAAAAAADAAAAAAAAAAAABAAAAAAAAAAAAAAAAAAAAAQAAAAAAAEAAAAAAAAAAQAEAAAAAAABwAAAAAAAAALABAAAAAAAAAAAAAAAAAACwAQAAAAAAAEAAAAAAAAAA8AEAAAAAAACYAAAAAAAAAAAAAAADAAAADwAAAAAAAAAAAAAAAAAAAA8AAAAAAAAAAAAAAAAAAAAPAAAAAAAAAAAAAAAAAAAAAAAAABEAAAAiAAAALwAAADsAAABFAAAAUAAAAFQAAABaAAAAXgAAAGsAAACAAAAAlQAAAKsAAAC3AAAAuwAAAGF2YWlsYWJpbGl0eV96b25lbWljcm9zZXJ2aWNlX25hbWVpbnN0YW5jZV9uYW1lcHJvY2Vzc19uYW1lb3NfdmVyc2lvbmpka192ZXJzaW9uY2VsbHJlZ2lvbnNpbG9pbnN0YW5jZV90eXBlbWVhc3VyZV92YWx1ZTo6ZG91YmxlbWVhc3VyZV92YWx1ZTo6YmlnaW50bWVhc3VyZV92YWx1ZTo6dmFyY2hhcm1lYXN1cmVfbmFtZXRpbWUAAAAAAAAAAAAHAAAADgAAABUAAAAcAAAAIwAAACoAAAAxAAAAOAAAAD8AAABGAAAATAAAAFIAAABZAAAAYAAAAGkAAAB2YXJjaGFydmFyY2hhcnZhcmNoYXJ2YXJjaGFydmFyY2hhcnZhcmNoYXJ2YXJjaGFydmFyY2hhcnZhcmNoYXJ2YXJjaGFyZG91YmxlYmlnaW50dmFyY2hhcnZhcmNoYXJ0aW1lc3RhbXAAAAAAAAAAAAAAAAkAAAASAAAAGwAAACQAAAAtAAAANgAAAD8AAABIAAAAUQAAAFoAAABnAAAAdAAAAIEAAACNAAAAlgAAAERJTUVOU0lPTkRJTUVOU0lPTkRJTUVOU0lPTkRJTUVOU0lPTkRJTUVOU0lPTkRJTUVOU0lPTkRJTUVOU0lPTkRJTUVOU0lPTkRJTUVOU0lPTkRJTUVOU0lPTk1FQVNVUkVfVkFMVUVNRUFTVVJFX1ZBTFVFTUVBU1VSRV9WQUxVRU1FQVNVUkVfTkFNRVRJTUVTVEFNUAAAEAAAAAwAFAASAAwACAAEAAwAAAAQAAAALAAAADwAAAAAAAMAAQAAAPgBAAAAAAAAIAEAAAAAAACIAgAAAAAAAAAAAAAAAAAAAAAAAAAACgAMAAAACAAEAAoAAAAIAAAAdAAAAAMAAABMAAAAKAAAAAQAAACg/v//CAAAAAwAAAAAAAAAAAAAAAUAAAByZWZJZAAAAMD+//8IAAAADAAAAAAAAAAAAAAABAAAAG5hbWUAAAAA4P7//wgAAAAMAAAAAgAAAHt9AAAEAAAAbWV0YQAAAAADAAAA3AAAAHAAAAAEAAAAQv///xQAAABEAAAARAAAAAAABQFAAAAAAQAAAAQAAAAw////CAAAABgAAAAPAAAAQ29sdW1uIENhdGVnb3J5AAQAAABuYW1lAAAAAAAAAAAw////DwAAAENvbHVtbiBDYXRlZ29yeQCq////FAAAADwAAAA8AAAAAAAFATgAAAABAAAABAAAAJj///8IAAAAEAAAAAQAAABUeXBlAAAAAAQAAABuYW1lAAAAAAAAAACQ////BAAAAFR5cGUAABIAGAAUABMAEgAMAAAACAAEABIAAAAUAAAARAAAAEgAAAAAAAUBRAAAAAEAAAAMAAAACAAMAAgABAAIAAAACAAAABAAAAAGAAAAQ29sdW1uAAAEAAAAbmFtZQAAAAAAAAAABAAEAAQAAAAGAAAAQ29sdW1uAAAYAgAAQVJST1cx
